src/tests/login.rs

```diff
@@ -82,7 +82,12 @@ fn test_login() {
     let mut token_info = CK_TOKEN_INFO::default();
     let ret = fn_get_token_info(testtokn.get_slot(), &mut token_info);
     assert_eq!(ret, CKR_OK);
-    assert_eq!(token_info.flags & pin_flags_mask, CKF_USER_PIN_COUNT_LOW);
+    if testtokn.dbtype == "nssdb" {
```

Comment:
Here too, I would just skip the operation, rather than matching to a fixed value.